### ISSUE TITLE
how to get this right

### ISSUE THREAD

**Issue by zorbahenry** [Microsoft/charticulator#156]
[flip.zip](https://github.com/Microsoft/charticulator/files/2892234/flip.zip)
I am following this demo:
https://charticulator.com/gallery/stock_msft.html 
but fail to get it right, here attached my CSV. and screen recording FYI. thanks

**Comment by snapperlux** [comment]
i agree the example is a little deceiving. The example image shows data on the Y-axis but you don't drop anything on the Y-axis in the video. The only way i could recreate this is by using a data axis in the glyph and mapping "start price" and "end Price" to it.

![screenshot 32](https://user-images.githubusercontent.com/43861167/53219486-f63dc580-3614-11e9-91d1-f63502004898.png)

**Comment by zorbahenry** [comment]
agree. thanks

在 2019年2月22日，下午12:12，snapperlux <notifications@github.com<mailto:notifications@github.com>> 写道：


i agree the example is a little deceiving. The example image shows data on the Y-axis but you don't drop anything on the Y-axis in the video. The only way i could recreate this is by using a data axis in the glyph and mapping "start price" and "end Price" to it.

[screenshot 32]<https://user-images.githubusercontent.com/43861167/53219486-f63dc580-3614-11e9-91d1-f63502004898.png>

**Comment by zorbahenry** [comment]
agree
thanks

**Comment by donghaoren** [comment]
Here is the problem: when the "stacking" scaffold is used, it's going to put the beginning of the first bar to the bottom, and the ending of the last bar to the top. However, your data sums to zero, this means the beginning of the first bar and the ending of the last bar should be the same. This creates a conflict. This is an issue by design so we can't easily fix it. @snapperlux's solution would be the best alternative way to go with.

We've just fixed a bug in #158, now if the data doesn't sum to zero (like the Microsoft Stock price data), then the method in the stock_msft demo should work.

**Comment by snapperlux** [comment]
Thanks for following up!

**Comment by nadiafank** [comment]
I have (I think) the same problem but I can't apply your solution. The bars don't paint from the Y-axis. Any orientation? 
Thanks for your help. 


<img width="1401" alt="charticulator-axis" src="https://user-images.githubusercontent.com/1250376/74439903-543e6b00-4e6d-11ea-8c31-ec839c4dc989.png">

![bar_charticulator_error](https://user-images.githubusercontent.com/1250376/74438764-27895400-4e6b-11ea-85e8-ab03eeffb1cf.jpg)
